packages/contracts/contracts/core/BaseVault.sol

```diff
@@ -51,6 +51,7 @@ contract BaseVault is ReentrancyGuard, Ownable, ERC20, Initializable {
     uint public managementFee;
 
     // Gap is left to avoid storage collisions. Though RibbonVault is not upgradeable, we add this as a safety measure.
+    // TODO: Do we need this?
```

Comment:
// It doesn't hurt, lot of OZ contract use it but we can delete it. I dont think it changes anything

Reply:
ok, let's keep it than. I just don't understand the reasoning :)

Reply:
https://forum.openzeppelin.com/t/what-exactly-is-the-reason-for-uint256-50-private-gap/798